lms/djangoapps/grades/api/urls.py

```diff
@@ -18,4 +18,25 @@
         ),
         views.CourseGradingPolicy.as_view(), name='course_grading_policy'
     ),
+    url(
+        r'^v0/course_grade/{course_id}/reportbysection/$'.format(
+            course_id=settings.COURSE_ID_PATTERN,
+        ),
+        views.GradeReportBySection.as_view(), name='grade_course_report_by_section'
+    ),
+    url(r'^v0/course_grade/report/(?P<uuid>[-\w]+)/$',
+        views.GradeReporByTaskId.as_view(), name='grade_course_report_generated'
+    ),
+    url(
+        r'^v0/course_grade/{course_id}/reportbyassignmenttype/$'.format(
+            course_id=settings.COURSE_ID_PATTERN,
+        ),
+        views.GradeReportByAssignmentType.as_view(), name='grade_course_report_by_assignment_type'
+    ),
+    url(
+        r'^v0/course_grade/{course_id}/reportenhancedproblemgrade/$'.format(
```

Comment:
split url name from `reportenhancedproblemgrade` to `report_enhanced_problem_grade`